src/SearchResults/generate-hierarchical-tree.js

```diff
@@ -82,7 +82,9 @@ function generateHierarchicalTree(
          * @type {object[]]} hierarchical data
          */
         var data = parent && Array.isArray(parent.data) ? parent.data : [];
-        parent = find(data, {isRefined: true});
+        parent = find(data, function(subtree) {
+          return subtree.isRefined;
```

Comment:
The two are not strictly equivalent, but I guess it's fine since we should control the `isRefined` value.